public/app/percona/shared/components/Elements/FeatureLoader/FeatureLoader.types.ts

```diff
@@ -3,4 +3,5 @@ import { Settings } from 'app/percona/settings/Settings.types';
 export interface FeatureLoaderProps {
   featureName: string;
   featureFlag: keyof Settings;
+  onError?: (error: any) => void;
```

Comment:
Can you try to use `Error` type here?

Reply:
The `Error` type does not have `response` field, which is what's needed on `CheckPanel`, for example. I used `AxiosError`, which is what we're looking for, since this error comes from the API call.

Reply:
That's even better!